Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM nfcore/base
-LABEL description="Docker image containing all requirements for nf-core/epitopeprediction pipeline"
+FROM nfcore/base:1.7
+LABEL authors="Christopher Mohr, Alexander Peltzer" \
+      description="Docker image containing all software requirements for the nf-core/epitopeprediction pipeline"
 
+# Install the conda environment
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-epitopeprediction-1.0dev/bin:$PATH
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/nf-core-epitopeprediction-1.0.0/bin:$PATH
+
+# Dump the details of the installed packages to a file for posterity
+RUN conda env export --name nf-core-epitopeprediction-1.0dev > nf-core-epitopeprediction-1.0dev.yml
```

Comment:
```suggestion
RUN conda env export --name nf-core-epitopeprediction-1.0.0 > nf-core-epitopeprediction-1.0.0.yml
```

Reply:
But do you need that line anyways? Should be already in github, no?

Reply:
We kind of agreed in the nf-core/tools template to have this in all pipelines from now on 👍